### ISSUE TITLE
First tab selects wrong element 

### ISSUE THREAD

**Issue by kvaternio-dev** [Bkucera/cypress-plugin-tab#2]
I tried the cypress-plugin-tab, I have some differences in order at manual testing, if I have divs in body, which have different tabindices, e.g. this one:

```
<html>
  <head>
  </head>
  <body>
    <div tabindex="10">1</div>
    <div tabindex="5">2</div>
    <div tabindex="3">3</div>
  </body>
</html>
```

When using the plugin, the first cy.get("body").tab() selects the div with "10" tabindex.
But at manual testing the "3" tabindex won.

Is it a bug of the plugin, or I did something wrong? How can I fix that?
It seems to me alphabetical order instead of numerical order.

**Comment by Bkucera** [comment]
@kvaternio-dev yes, this is probably a bug with the plugin. _However_ MDN warns against using any value greater than 0 as a tabindex, as it should follow the relative order of DOM nodes. https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex

**Comment by seanforyou23** [comment]
+1 to this issue. Even though it's not recommended practice, I'd like to think a plugin that's all about testing tab functionality handles the most basic case of simply loading a page and pressing the tab key.

We're building a framework of UI components over at PatternFly, and even though we also don't recommend using positive values, we'd like to support the capability for advanced used cases. To verify that the positive `tabindex` values are respected, I was hoping to be able to write tests using this plugin that exercise pretty much exactly the code path that @kvaternio-dev mentioned in this issue.

Do you think we could consider addressing this at some point? Seems like it wouldn't be too bad, just build a map and sort by tabindex value, totally just speculating though :) Thanks!

**Comment by Bkucera** [comment]
@seanforyou23 yes, I'll look into it

**Comment by seanforyou23** [comment]
I played with it a bit more, and you know, it actually [seems to be working](https://github.com/seanforyou23/patternfly-react/commit/e1cab9ebf2d28f4ced397bb68a3b90891fa6bf41#diff-ad8dcb99e87958a35e5815311202b43dR20-R29) for me now. It may have been working before, but an ide error #19 was throwing me off.

**Comment by Bkucera** [comment]
@seanforyou23 hmm...enough confusing me for the day :smile:

**Comment by seanforyou23** [comment]
Sorry bout that! 😅Just wanted to share that I had found some success in case it helps
